src/core/MacPasteboard.h

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright (C) 2017 KeePassXC Team <team@keepassxc.org>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 or (at your option)
+ *  version 3 of the License.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef KEEPASSXC_MACPASTEBOARD_H
+#define KEEPASSXC_MACPASTEBOARD_H
+
+#include <QMacPasteboardMime>
+#include <QTextCodec>
+
+class MacPasteboard : public QMacPasteboardMime
+{
+public:
+    explicit MacPasteboard() : QMacPasteboardMime(MIME_ALL) {}
+
+    QString convertorName();
```

Comment:
Overriding methods should be marked with `override`.

Reply:
@phoerious done